Bottom: d901bff8f644702c559d4ff810b2e8d765d3b3b6
Top:    0d446e5390746d812ba186cb3233366b0be4c260
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-01 23:43:00 +0000

hid/gcode: Add #include for locale.h required for setlocale()


---

diff --git a/src/hid/gcode/gcode.c b/src/hid/gcode/gcode.c
index 9a0894f..208d02d 100644
--- a/src/hid/gcode/gcode.c
+++ b/src/hid/gcode/gcode.c
@@ -45,6 +45,10 @@
 
 #include <time.h>
 
+#ifdef HAVE_LOCALE_H
+#include <locale.h>
+#endif
+
 #include "global.h"
 #include "error.h" /* Message() */
 #include "data.h"
